velox/exec/tests/SortBufferTest.cpp

```diff
@@ -64,31 +66,37 @@ class SortBufferTest : public OperatorTestBase,
         0,
         0,
         "none",
-        spillPrefixSortConfig_);
+        withPrefixSort ? spillPrefixSortConfig_ : spillNonPrefixSortConfig_);
   }
 
   const bool enableSpillPrefixSort_{GetParam()};
   const velox::common::PrefixSortConfig prefixSortConfig_ =
-      velox::common::PrefixSortConfig{std::numeric_limits<int32_t>::max(), 130};
+      velox::common::PrefixSortConfig{
+          std::numeric_limits<int32_t>::max(),
+          130,
+          0.3};
   const std::optional<common::PrefixSortConfig> spillPrefixSortConfig_ =
       enableSpillPrefixSort_
       ? std::optional<common::PrefixSortConfig>(prefixSortConfig_)
       : std::nullopt;
 
+  const velox::common::PrefixSortConfig nonPrefixSortConfig_ =
```

Comment:
This config is same with prefixSortConfig_, does it really make sense. The nonPrefixSortConfig should fallback to stdsort